packages/core/docs/integrations/adyen.md

```diff
@@ -168,8 +168,36 @@ export default {
 
 Read [Adyen's document about the Klarna](https://docs.adyen.com/payment-methods/klarna#supported-countries) to check which Klarna payment methods are available for individual countries.
 
+:::warning Email Address
+Klarna requires shopper's email address to work correctly. It will be read from `cart.customerEmail` with fallback to the `cart.customer.email` field. Make sure to put it in one of these.
+:::
+
 :::warning Phone number
-If your users can provide a phone number, then make sure it is with **area code**. Otherwise, Klarna will throw an error because of an improper phone number format.
+If your users can provide a phone number then make sure it is with **area code**. Otherwise, Klarna will throw an error because of an improper phone number format.
```

Comment:
```suggestion
If your users provide a phone number, make sure it includes the **area code**. Otherwise, Klarna will throw an error because of the improper phone number format.
```